### PR TITLE
Replace same variables

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Currently, we have primitive colors with the same values. In this update, we consolidate them into a single value and replace references in the themes with non-deprecated ones that share the same value. None of them are used in the nns-dapp repo.

# Changes

- Update CSS variables.

### CODE REVIEW

File: src/lib/styles/global/colors.scss [hunk 1]
```diff
@@ -11,26 +11,27 @@
...
   --cp-light-250: #6f7fcd;
   // @deprecated
   --cp-light-400: #34407c;
   // @deprecated (same as --cp-light-600)
-  --cp-light-500: #3d4d99;
+  --cp-light-500: var(--cp-light-600);
   --cp-light-600: #3d4d99;
   --cp-light-900: #151a33;
-  // @deprecated
+  // @deprecated TODO: replace with Neutral-50A45
```
Comment (dskloetd) [inline src/lib/styles/global/colors.scss:23]: What is `Neutral-50A45`?

Comment (mstrasinskis) [reply src/lib/styles/global/colors.scss:23]: A new name for the `--cp-light-hover-overlay` variable.

Comment (dskloetd) [reply src/lib/styles/global/colors.scss:23]: But I don't see where it's defined.
Also for `--cp-light-hover-overlay` it says `same as --cp-light-50-45` so should that also say to use Neutral-50A45 instead?
And why such a strange name? It seems like a combination between a name and a color code. Shouldn't it be either/or?
What does 50A45 stand for?
And can we stick to lower kebab case for these variables?

Comment (mstrasinskis) [reply src/lib/styles/global/colors.scss:23]: It’s a CUSTOM adaptation of the Material Design approach by our designers, where the numbers represent the shade or intensity of the color. The `A` stands for alpha.

Comment (dskloetd) [reply src/lib/styles/global/colors.scss:23]: No, we don't use underscores anywhere in this file. And neither do we use upper case.
It's all a combination of lower case and hyphens. Like `--cp-light-hover-overlay`.
Unless you plan to switch everything to title case, why would we diverge from what we currently do?

And again, can you tell me where it's defined in gix-components?

Comment (mstrasinskis) [reply src/lib/styles/global/colors.scss:23]: Got it, sorry for the misunderstanding. I pasted the color name from Figma as a note to myself to remember to replace it. The name in the front-end will be `--neutral-50A45`.

**Review by dskloetd — APPROVED** [review]
I still don't get it but I guess you're removing it anyway in the next PR.

**Pull request merged**